dvm/commands/Install.d

```diff
@@ -137,7 +137,19 @@ private:
             wrapper.path ~= ".bat";
 
         verbose("Installing wrapper: " ~ wrapper.path);
-        wrapper.write;
+        try
+        {
+            wrapper.write;
+        }
+        catch (Exception ex)
+        {
+            import std.regex;
+            if (matchFirst(ex.toString(), r"dmd-[\d\.]+\.bat"))
```

Comment:
I don't like using the textual representation of an exception to identify what kind of exception it is. Please either catch the specific exception type or make an explicit check that the necessary file/directory structure is available.

Perhaps it's worth adding a more general check which is executed for all commands?